packages/actor-query-operation-orderby-sparqlee/lib/ActorQueryOperationOrderBySparqlee.ts

```diff
@@ -25,7 +25,8 @@ export class ActorQueryOperationOrderBySparqlee extends ActorQueryOperationTyped
     // Will throw error for unsupported operators
     for (let expr of pattern.expressions) {
       expr = this.extractSortExpression(expr);
-      const _ = new AsyncEvaluator(expr);
+      const sparqleeConfig = { ...ActorQueryOperation.getAsyncExpressionContext(context) };
+      const _ = new AsyncEvaluator(expr, sparqleeConfig);
```

Comment:
Any reason to not just do this?:
```suggestion
      const _ = new AsyncEvaluator(expr, ActorQueryOperation.getAsyncExpressionContext(context));
```

Reply:
The tests needs to check whether the actor can execute the desired action. The actor answers this request by asking sparqlee. If a extension function is present within the expression, sparqlee will answer `no` unless it is gets that extension function wiht the `ExtensionFunctionCreator`. It is thus needed to provide the context (or at least the ExtensionFunctionCreator) here.

Reply:
Should I add a comment here?

Reply:
Sure, I get that. My comment was just about inlining. Since there is no need to have a dedicated `sparqleeConfig` AFAICS.